vars/Debian.yml

```diff
@@ -10,5 +10,5 @@ __apache_packages:
   - apache2-utils
 
 apache_ports_configuration_items:
-  - regexp: "^Listen "
+  - regexp: "^Listen {{ apache_listen_port }}$"
```

Comment:
One issue here—this would break the functionality of _changing_ any Listen directive...